components/Accordion/src/index.scss

```diff
@@ -117,18 +120,30 @@
   --utrecht-heading-font-weight: var(--denhaag-accordion-container-open-title-font-weight);
 }
 
-.denhaag-accordion__container--open .denhaag-accordion__panel:hover,
 .denhaag-accordion__container--open .denhaag-accordion__panel > .denhaag-icon:hover {
   --denhaag-accordion-title-color: var(--denhaag-accordion-container-open-color);
 }
 
+.denhaag-accordion__container--open .denhaag-accordion__title:not([disabled]) {
+  --utrecht-heading-5-color: var(--denhaag-accordion-panel-color);
+  --utrecht-heading-5-font-weight: 700;
+}
```

Comment:
If you want to force the heading of the accordion to have a certain style, and override the heading default styling, I suggest two options:

- do not use `utrecht-heading-*` at all, just use `<h2 class="denhaag-accordion_title">`-`<h6>`
- do not only style `--utrecht-heading-5`, but also all the tokens for `utrecht-heading-2`/`3`/`4`/`6`. Accordions can occur at any level in the page and should not be hardcoded to `h3` or `h5`. That would be a WCAG violation.

Reply:
@Robbert , What about the element can change from heading-size but the CSS selector stays the same on utrecht-heading-5?

Reply:
That is fine too from an accessibility standpoint, but what would be the advantage? Still seems like an odd choice to use the `heading-5` element to style the accordion heading, instead of the other way around, if you want to apply some of the design choices from heading 5.

You could do:

```json
{
  "denhaag": {
    "accordion": {
      "title": { "font-size": { "value": "{utrecht.heading-5.font-size}" } }
    }
  }
}
```

Reply:
The only advantage would be less bloating in duplicate styles